sdks/python/apache_beam/examples/ml-orchestration/kfp/requirements.txt

```diff
@@ -0,0 +1,2 @@
+kfp==1.8.13
+google-cloud-aiplatform==1.15
```

Comment:
I may have missed this, but is it explained how this file is used?

Reply:
It's not explicitly explained no. Do you think it might be unclear that these are requirements to compile and run the kfp pipeline? If so, where would you suggest to put the extra explanation? It might be a little too detailed for the conceptual guide in the docs.

Reply:
Personally, I was able to infer that, but this may not be clear to everybody. Can we add a comment in this file, and also do a short explanation in the docs? Maybe in the section where we talk about executing the KFP pipeline?